### PR TITLE
Added gaia worker documentation

### PR DESCRIPTION
**Pull request by michelvocks**
Related PR: https://github.com/gaia-pipeline/gaia/pull/166

### CODE REVIEW

**Comment by Skarlso** [comment]
Oh cool! Been waiting fro this. :D I will meticulously read it. :P

**Comment by Skarlso** [comment]
Oh what's hard to notice here is that I fixed a few typos and reworded a few things here and there.

But the damn indentation messed up the difference. :/ Sorry about that man. I thought it would be better visible that way.

**Review by Skarlso — APPROVED** [review]

File: content/operations/worker/_index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+## Gaia Worker 
+
+Scalability is one of the leading biggest issues around CI/CD and automation. Often, it is not trivial to scale pipelines
+horizontally whereas vertical scaling works until a specific point has been reached. Gaia Worker helps you to solve this
+problem by providing a remote worker functionality for your Gaia primary instance.
+
+Gaia Workers allow you to dynamically add or remove remote workers to your primary Gaia instance and tag those differently.
+Once a worker has been registered at your primary Gaia instance, new pipeline runs will be automatically be scheduled to
```
Comment (michelvocks) [inline content/operations/worker/_index.md]:
```suggestion
Once a worker has been registered at your primary Gaia instance, new pipeline runs will be automatically scheduled to
```

File: content/operations/worker/_index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+your worker by matching pipeline tags and worker tags.
+
+While your pipeline run is executed at the remote worker, all pipeline run information like current job status, log
+output and run status, is transferred in real-time to the primary Gaia instance and can be observed via the UI.
+
+The scheduler from the primary Gaia instance automatically monitors and detects the status from remote workers. If a
+worker goes down or loses the connection to the primary Gaia instance, it will automatically be flagged as inactive and
+no further pipeline runs will be scheduled on this node till the node is active again. Additionally, the primary Gaia
```
Comment (michelvocks) [inline content/operations/worker/_index.md]:
```suggestion
no further pipeline runs will be scheduled to this node till the node is active again. Additionally, the primary Gaia
```

File: content/rfcs/0001_distributed_execution_of_pipelines.md [hunk 3]
```diff
@@ -84,85 +83,128 @@ The workers will need to be tagged with what kind of resource they are providing
...
+    repeated string tags  = 3;
+}
+
+// PipelineRun represents one pipeline run.
+message PipelineRun {
+    string   unique_id     = 1;
+    int64    id            = 2;
+    string   status        = 3;
+    int64    start_date    = 4;
+	int64    finish_date   = 5;
```
Comment (Skarlso) [inline content/rfcs/0001_distributed_execution_of_pipelines.md]: There are some very weird indentations in this. :D Can you fix that ? After that it's cool. :)

**Review by Skarlso — APPROVED** [review]

**Comment by Skarlso** [comment]
👍 Good stuff man

**Comment by Skarlso** [comment]
@michelvocks This can be merged, right?

**Comment by michelvocks** [comment]
@Skarlso Yeah I wanted to merge that with the main PR https://github.com/gaia-pipeline/gaia/pull/166
Just waiting for your final approval 😄

**Pull request merged**